SwanCustomEnvironments/swancustomenvironments/scripts/builders/venv.sh

```diff
@@ -16,14 +16,14 @@ eval "${ACTIVATE_ENV_CMD}"
 # Otherwise, use pip for resolution (more reliable long-term).
 _log "Installing packages from ${REQ_PATH}..."
 if [ "${RESOLVED_REQ}" = true ]; then
-    uv pip install -r "${REQ_PATH}" 2>&1
+    uv pip install ${R_FLAG} "${REQ_PATH}" 2>&1
     if [ $? -ne 0 ]; then
         return 1
     fi
     # Enforce installation of our version of ipykernel and its dependencies
     uv pip install ${IPYKERNEL} 2>&1
 else
-    pip install -r "${REQ_PATH}" 2>&1
+    pip install ${R_FLAG} "${REQ_PATH}" 2>&1
```

Comment:
So iiuc uv pip install and pip install, when receiving a directory as argument, they look for a pyproject.toml in it?